acceptance/indy_acceptance/indy-tests/test_results/Acceptance_Test_20171117_17-42-58/Test_Scenario_11_Special_Case_Trust_Anchor_Role_20171117_17-42-58.json

```diff
@@ -0,0 +1,98 @@
+{
```

Comment:
It looks like a log. I think it needs to be removed

Reply:
We removed it.